### PR TITLE
fixes #16151 - strong params support for foreman_chef

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by stbenjam** [comment]
Still a WIP, but have a look if you'd like.  I haven't been able to test it since my dev box isn't working after webpack changes.

File: lib/foreman_chef/engine.rb [hunk 1]
```diff
@@ -61,6 +61,11 @@ class Engine < ::Rails::Engine
              caption: N_('Environments'),
              parent: :configure_menu,
              last: true
+
+
```
Comment (ares) [inline lib/foreman_chef/engine.rb]: could you please remove extra empty lines here? one should be fine

**Comment by ares** [comment]
It seems to work well, I'll keep it around for a while to see if everything works as expected. Could you please use [#16151](http://projects.theforeman.org/issues/16151) for this? One huge thank you for this.

**Comment by stbenjam** [comment]
@ares Sure, done.  I fixed my dev environemnt (foreman_chef needed the initializer hook changed to before instead of after), and this all seems to work.  

Would greatly appreciate a release once this is merged, as it's holding up foreman-tasks update to fix Katello BATS (https://github.com/theforeman/foreman-packaging/pull/1265).

**Comment by ares** [comment]
sure thing, I hope to do the release later today

**Comment by ares** [comment]
@stbenjam sorry I fixed the initializer separately, could you please remove/rebase it? good to merge right after

**Comment by ares** [comment]
actually the gemspec patch was also merged, rebase should solve it

**Comment by ares** [comment]
I rebased and cherrypicked as 97fe5f3bd5deae564e6b2a717cb4de63071d9b1e, closing this. Thank you @stbenjam!

**Comment by stbenjam** [comment]
Thanks!

**Pull request closed without merging**